### PR TITLE
[otl] Upgrade to version 4.0.448

### PR DESCRIPTION
**Pull request by LilyWangL**
Related issue: #8794. There are no features of this port need to test.

### CODE REVIEW

**Review by NancyLi1013 — APPROVED** [review]

File: ports/otl/portfile.cmake [hunk 1]
```diff
@@ -2,8 +2,8 @@ include(vcpkg_common_functions)
 
 vcpkg_download_distfile(ARCHIVE
     URLS "http://otl.sourceforge.net/otlv4_h2.zip"
```
Comment (ras0219-msft) [inline ports/otl/portfile.cmake]: We should use a stable link: http://otl.sourceforge.net/otlv4_40448.zip

**Review by ras0219-msft — CHANGES_REQUESTED** [review]

File: ports/otl/portfile.cmake [hunk 2]
```diff
@@ -12,5 +12,5 @@ vcpkg_extract_source_archive_ex(
     NO_REMOVE_ONE_LEVEL
 )
 
-file(INSTALL ${SOURCE_PATH}/otlv4.h DESTINATION ${CURRENT_PACKAGES_DIR}/include/otl)
-file(INSTALL ${SOURCE_PATH}/otlv4.h DESTINATION ${CURRENT_PACKAGES_DIR}/share/otl RENAME copyright)
+file(INSTALL ${SOURCE_PATH}/otlv40448.h DESTINATION ${CURRENT_PACKAGES_DIR}/include/otl)
```
Comment (ras0219-msft) [inline ports/otl/portfile.cmake]: We should either rename this file to `otlv4.h` or create a file called `otlv4.h` that is just `#include <otlv40448.h>`

**Review by ras0219-msft — CHANGES_REQUESTED** [review]

**Pull request merged**